src/main/java/com/game/roullet/service/RoomService.java

```diff
@@ -51,4 +53,37 @@ public int createRoom(int playerId) {
     }
 
 
+    public JoinResponse joinRoom(int playerId, int roomId) {
+
+        JoinResponse joinResponse = new JoinResponse();
+        joinResponse.setRoomStatus(false);
+        List<Registration> registrationList = registrationRepository.findAllRegistrationByRoomId(roomId);
```

Comment:
You should not do this call in front. Because few lines down you just do a validation to see if the player and the room exist.
If the player or the room does not exist in this case your call to the database **does not make sense**. You could improve you performance just b_y moving this line  down_ to a properly place.